testing/test-helper.js

```diff
@@ -277,6 +277,12 @@ export class BrowserController {
     }
   }
 
+  clickAtPosition(selector, clientX = 0, clientY = 0) {
+    const element = this.doc_.querySelector(selector);
+    const clickEvent = new MouseEvent('click', {clientX, clientY});
+    element.dispatchEvent(clickEvent);
```

Comment:
Does it work if we do `this.doc_.dispatchEvent` so we don't have to pass the `selector` attribute and can only give the positions?

Reply:
Unfortunately not. I played around with it and tried doing `this.doc_.body.dispatchEvent`, but since the event stays at the `<body>` level, the story never receives the event and thus cannot act upon it. Let me know if you have any other ideas.

Reply:
Oh, got it. The helper name is maybe a bit confusing then. The set of argument is maybe too specific to your use case to be useful to others. If there is no way to get it right (ideally, only use clientX/clientY), I think we should keep this helper at the `test-amp-story-analytics` level.